src/renderer/src/routes/pages/Swap/SwapPageContent.tsx

```diff
@@ -417,15 +418,16 @@ export const SwapPageContent = ({ account }: TProps) => {
         let totalFeeAmount = NumberHelper.number(fee)
 
         if (
-          UtilsHelper.normalizeHash(actionData.selectedTokenToUse.value?.hash) ===
-          UtilsHelper.normalizeHash(service.feeToken.hash)
+          BSTokenHelper.normalizeHash(actionData.selectedTokenToUse.value?.hash) ===
+          BSTokenHelper.normalizeHash(service.feeToken.hash)
         ) {
           totalFeeAmount += NumberHelper.number(actionData.selectedAmountToUse.value)
         }
 
         const feeBalanceNumber =
           balanceQuery.data?.tokensBalances.find(
-            ({ token }) => UtilsHelper.normalizeHash(token.hash) === UtilsHelper.normalizeHash(service.feeToken.hash)
+            ({ token }) =>
+              BSTokenHelper.normalizeHash(token.hash) === BSTokenHelper.normalizeHash(service.feeToken.hash)
```

Comment:
Create a variable with this value: `BSTokenHelper.normalizeHash(service.feeToken.hash)`.